### ISSUE TITLE
Entities can't save both a skin and render type because...

### ISSUE THREAD

**Issue by SuprD** [zhuowei/MCPELauncher#849]
...BlockLauncher only saves one piece of extra data per entity. It only saves the most recently added extra data for that entity, for example only the custom render. If you add your own extra data beyond that, only the new one is saved and neither skin nor render type are saved.

**Comment by MooGaming227** [comment]
use entity added hook and just say if entity has this skin, do that.

**Comment by SuprD** [comment]
Yeah, that's a workaround, but the fact that entities only save one piece of extra data is still a bug.
